Wikipedia/Code/PlacesViewController.swift

```diff
@@ -661,6 +672,15 @@ class PlacesViewController: UIViewController, MKMapViewDelegate, UISearchBarDele
                 
                 guard result.error == nil else {
                     WMFAlertManager.sharedInstance.showWarningAlert(result.error!.localizedDescription, sticky: false, dismissPreviousAlerts: true, tapCallBack: nil)
+                    
+                    let error: Error = result.error!
```

Comment:
I see that this is in the "else" condition of `== nil` but maybe instead it's just `if let error = ` with an early return inside the if